shell/platform/tizen/tizen_renderer.h

```diff
@@ -29,6 +29,7 @@ class TizenRenderer {
 
   bool IsValid() { return is_valid_; }
 
+  virtual void* GetWindow() = 0;
```

Comment:
I just realized that there's already `GetWindowHandle`.

Reply:
yes, but EvasGL renderer return nullptr.
Hi,@bwikbs do we could  return evas_window_ by GetWindowHandle?

Reply:
@xiaowei-guan 
Sorry for late reply!
> Hi,@bwikbs do we could return evas_window_ by GetWindowHandle?

Of course it is possible! :smil

Reply:
ok,Thank you~